x-pack/legacy/plugins/code/server/routes/redirect.ts

```diff
@@ -1,39 +0,0 @@
-/*
```

Comment:
@spacedragon this file does not seem to be used anywhere, so remove now.

Reply:
Sure, thanks.